server/controllers/list.js

```diff
@@ -505,6 +505,7 @@ const restoreList = async (req, resp) => {
       .populate('viewersIds', 'avatarUrl displayName _id')
       .populate('items.authorId', 'displayName')
       .populate('items.editedBy', 'displayName')
+      .populate('cohortId', 'name memberIds ownerIds')
```

Comment:
this line is unneeded because you get for the cohort in a separate query.

Reply:
You're wrong I need them.
without this population line it doesn't fix anything